### PR TITLE
Update upgrade doc

### PR DESCRIPTION
**Pull request by blacksponge**
## Purpose

Add missing instruction in upgrade document

## Changes

* clean old patch files
* fix path to patch for sqlite
* fix mysql upgrade

## How to test this PR

follow upgrade document

### CODE REVIEW

**Review by pnax — DISMISSED** [review]

File: docs/upgrade/upgrade_zonemaster_backend_ver_8.0.0.md [hunk 1]
```diff
@@ -75,6 +75,14 @@ install -v -m 755 ./zm_rpcapi-bsd /usr/local/etc/rc.d/zm_rpcapi
 ```
 
 
+## Cleaning old files
+
+Run
+```sh
+cd $(perl -MFile::ShareDir -le 'print File::ShareDir::dist_dir("Zonemaster-Backend")')
+sudo rm patch_*.pl
+```
```
Comment (matsduf) [inline docs/upgrade/upgrade_zonemaster_backend_ver_8.0.0.md:84]: * `$(...)` works in Bash, Bourne shell and some other shells, but not in csh, the default FreeBSD root shell.
* sudo() does not work in FreeBSD.

````
Run in Linux:
```sh
sudo rm $(perl -MFile::ShareDir -le 'print File::ShareDir::dist_dir("Zonemaster-Backend")')/patch_*.pl
```
Run in FreeBSD:
```sh
rm `perl -MFile::ShareDir -le 'print File::ShareDir::dist_dir("Zonemaster-Backend")')`/patch_*.pl
```
````

File: docs/upgrade/upgrade_zonemaster_backend_ver_8.0.0.md [hunk 2]
```diff
@@ -90,12 +98,17 @@ than v8.0.0, and not upgraded, use the following instructions.
...
 ```sh
 cd $(perl -MFile::ShareDir -le 'print File::ShareDir::dist_dir("Zonemaster-Backend")')
-perl patch_sqlite_db_zonemaster_backend_ver_8.0.0.pl
+perl patch/patch_sqlite_db_zonemaster_backend_ver_8.0.0.pl
 ```
 
 ### MySQL (or MariaDB)
 
-Run
+First update the privileges of the `zonemaste` user:
```
Comment (matsduf) [inline docs/upgrade/upgrade_zonemaster_backend_ver_8.0.0.md:106]: "zonemaste" --> "zonemaster"

File: docs/upgrade/upgrade_zonemaster_backend_ver_8.0.0.md [hunk 3]
```diff
@@ -90,12 +98,17 @@ than v8.0.0, and not upgraded, use the following instructions.
...
+```sh
+sudo mysql -e "GRANT ALL ON zonemaster.* TO 'zonemaster'@'localhost';"
```
Comment (matsduf) [inline docs/upgrade/upgrade_zonemaster_backend_ver_8.0.0.md:108]: For FreeBSD:

```
mysql  -u root -p -e "GRANT ALL ON zonemaster.* TO 'zonemaster'@'localhost';"
```

**Review by matsduf — CHANGES_REQUESTED** [review]

File: docs/upgrade/upgrade_zonemaster_backend_ver_8.0.0.md [hunk 4]
```diff
@@ -90,12 +98,17 @@ than v8.0.0, and not upgraded, use the following instructions.
...
+```
+
+then run
 ```sh
 cd $(perl -MFile::ShareDir -le 'print File::ShareDir::dist_dir("Zonemaster-Backend")')
```
Comment (matsduf) [inline docs/upgrade/upgrade_zonemaster_backend_ver_8.0.0.md:113]: Also her there is `$(...)` that does not work with FreeBSD. Please use backtick instead. The same on 8 lines down.

File: docs/upgrade/upgrade_zonemaster_backend_ver_8.0.0.md (hunk 1 above)
Comment (blacksponge) [reply docs/upgrade/upgrade_zonemaster_backend_ver_8.0.0.md]: fixed

File: docs/upgrade/upgrade_zonemaster_backend_ver_8.0.0.md (hunk 2 above)
Comment (blacksponge) [reply docs/upgrade/upgrade_zonemaster_backend_ver_8.0.0.md]: fixed

File: docs/upgrade/upgrade_zonemaster_backend_ver_8.0.0.md (hunk 3 above)
Comment (blacksponge) [reply docs/upgrade/upgrade_zonemaster_backend_ver_8.0.0.md]: fixed

File: docs/upgrade/upgrade_zonemaster_backend_ver_8.0.0.md (hunk 4 above)
Comment (blacksponge) [reply docs/upgrade/upgrade_zonemaster_backend_ver_8.0.0.md]: fixed

**Review by mattias-p — APPROVED** [review]
LGTM

**Review by matsduf — APPROVED** [review]

**Review by matsduf — APPROVED** [review]
I approve.

**Pull request merged**